src/arrays/quarray.jl

```diff
@@ -34,6 +34,8 @@
     bases(qarr::QuArray, i) = qarr.bases[i]
     bases(qarr::AbstractQuArray) = ntuple(ndims(qarr), i->bases(qarr, i))
 
+    copy{B,T,N,A}(qa::QuArray{B,T,N,A}) = QuArray(copy(qa.coeffs), copy(qa.bases))
```

Comment:
Use `Base.copy{B,T,N,A}` instead to extend the function in Base and to make it visible outside QuBase. (I think you don't need the type parameters here, so `qa::QuArray` should be sufficient.